ShopifySharp/Services/AccessScope/AccessScopeService.cs

```diff
@@ -20,7 +22,31 @@ public class AccessScopeService : ShopifyService, IAccessScopeService
         public AccessScopeService(string myShopifyUrl, string shopAccessToken) : base(myShopifyUrl, shopAccessToken) { }
 
         /// <inheritdoc />
-        public virtual async Task<IEnumerable<AccessScope>> ListAsync(CancellationToken cancellationToken = default) => 
-            await ExecuteGetAsync<IEnumerable<AccessScope>>("oauth/access_scopes.json", "access_scopes", cancellationToken: cancellationToken);
+        public virtual async Task<IEnumerable<AccessScope>> ListAsync(CancellationToken cancellationToken = default)
+        {
+            return await ExecuteGetAsync<IEnumerable<AccessScope>>("oauth/access_scopes.json", "access_scopes", cancellationToken: cancellationToken);
+        }
+
+        /// <summary> 
+        ///  Requests a subset of granular access scopes for an individual shop installation. 
+        /// </summary> 
+        public virtual async Task<IEnumerable<AccessScope>> RequestGranularAccessScopesAsync(IEnumerable<AuthorizationScope> requestedScopes, CancellationToken cancellationToken = default) 
+        {
+            return await RequestGranularAccessScopesAsync(requestedScopes.Select(s => s.ToSerializedString()), cancellationToken); 
+        }
+
+        /// <summary> 
+        ///  Requests a subset of granular access scopes for an individual shop installation. 
+        /// </summary> 
+        public virtual async Task<IEnumerable<AccessScope>> RequestGranularAccessScopesAsync(IEnumerable<string> requestedScopes, CancellationToken cancellationToken = default)
```

Comment:
Same here, this just needs to be added to the IAccessScopeService interface and we'll be good to go!